src/components/Generic/FileExplorerSideNav.vue

```diff
@@ -616,7 +616,7 @@ export default defineComponent({
     clickSettings() {
       if (!this.storedSideNavPrinter) return;
       this.printersStore.setUpdateDialogPrinter(this.storedSideNavPrinter);
-      this.dialogsStore.openDialog(DialogName.UpdatePrinterDialog);
+      this.dialogsStore.openDialog(DialogName.CreatePrinterDialog);
```

Comment:
As the dialog will serve both updates as well as creates, I intend to rename this to just PrinterDialog.